conn.go

```diff
@@ -476,11 +490,91 @@ func (cn *conn) gname() string {
 	return strconv.FormatInt(int64(cn.namei), 10)
 }
 
-func (cn *conn) simpleExec(q string) (res driver.Result, commandTag string, err error) {
-	b := cn.writeBuf('Q')
-	b.string(q)
-	cn.send(b)
+func (cn *conn) awaitSynchronizationPoint() {
```

Comment:
This doesn't do much waiting. It looks a lot more like the read* methods (one or two expected messages, else errorf.)